domain/consensus/processes/pruningmanager/pruningmanager.go

```diff
@@ -255,9 +279,13 @@ func (pm *pruningManager) deletePastBlocks(stagingArea *model.StagingArea, pruni
 	if err != nil {
 		return err
 	}
-	err = queue.PushSlice(parents)
-	if err != nil {
-		return err
+
+	isChildOfVirtualGenesis := len(parents) == 1 && parents[0].Equal(model.VirtualGenesisBlockHash)
```

Comment:
USe `isVirtualGenesisOnlyParent` (It's in blockValidator rn, but I guess it can be moved somewhere more common)

Reply:
I prefer code duplication of the method, because adding it to utils feels weird.